.eslintignore

```diff
@@ -7,6 +7,8 @@ extension/options/routinesView.js
 extension/options/history/historyView.js
 extension/onboarding/onboardingController.js
 extension/onboarding/onboardingView.js
+extension/homepage/homepageController.js
+extension/homepage/homepageView.js
```

Comment:
I think this should all go in `homepage/` (besides, not in, `extension/`). And then we can also probably build it separately. Potentially we should even create `homepage/package.json` and `homepage/public/index.html` etc. Then the build steps can be different. But I'm also fine having the root `package.json` build it.